xarm:
  hardware_interface:
     loop_hz: 10 # hz
     joints:
       - xarm_6_joint
       - xarm_5_joint